lms/envs/production.py

```diff
@@ -599,6 +599,9 @@ def get_env_setting(setting):
     "MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS", MAX_FAILED_LOGIN_ATTEMPTS_LOCKOUT_PERIOD_SECS
 )
 
+##### LOGISTRATION RATE LIMIT SETTINGS #####
+LOGISTRATION_RATELIMIT_RATE = ENV_TOKENS.get('LOGISTRATION_RATELIMIT_RATE', LOGISTRATION_RATELIMIT_RATE)
```

Comment:
that code should also be added in cms/envs/production

Reply:
@adeelehsan No, we don't need it over there. We just need to add it here due to the errors while loading the CMS urls.